### PR TITLE
[intl-tel-input] Fix Global Scope for Utils

### PR DESCRIPTION
**Pull request by Crocsx**
Select one of these and delete the others:

If changing an existing definition:

    Provide a URL to documentation or source code which provides context for the suggested changes: <>

Missing definitions for global scope methods

https://github.com/jackocnr/intl-tel-input/wiki/utils.js

### CODE REVIEW

File: types/intl-tel-input/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for intl-tel-input 17.0
+// Type definitions for intl-tel-input 17.0.3
```
Comment (peterblazejewicz) [inline types/intl-tel-input/index.d.ts:1]:
```suggestion
// Type definitions for intl-tel-input 17.0
```
There is no patchin in DT SemVer:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Comment by peterblazejewicz** [comment]
Please add test to `*-global.ts` file to cover meethods, you can use code from wiki, e.g. that way:
```ts
// $ExpectType string
intlTelInputUtils.formatNumber("07733123456", "gb", intlTelInputUtils.numberFormat.INTERNATIONAL);
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Crocsx thx!

**Comment by Crocsx** [comment]
Sorry for the mess with the test ahah

**Comment by Crocsx** [comment]
Ready to merge

**Pull request merged**